tests/expose_test.go

```diff
@@ -720,8 +745,7 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				Eventually(endptSubsetIpAddress).Should(BeEquivalentTo(vmPodIpAddress))
 
 				By("Deleting the VM.")
-				Expect(virtClient.VirtualMachine(vm.Namespace).Delete(vm.Name, &k8smetav1.DeleteOptions{})).To(Succeed())
-				tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
+				Expect(cleanupVM(virtClient, vm.GetNamespace(), vm.GetName())).To(Succeed(), "the VM object should be deleted")
```

Comment:
Why is this special cleanup needed? The test is flaky if we don't wait for the vm object removal?